--- Makefile	2011-11-24 12:41:08.000000000 +0000
+++ Makefile.new	2018-10-07 11:01:11.959570491 +0100
@@ -4,7 +4,7 @@
 
 LEX = flex
 CXXFLAGS += -Wall -Wno-unused-function -g
-CXXFLAGS += $(shell pkg-config --cflags htmlcxx)
+CXXFLAGS += -I$(DIR_HTMLCXX)/include -fPIC
 LFLAGS += --reentrant
 
 all: lib
@@ -20,5 +20,8 @@
 
 lib: lexer.h libhcxselect.a
 
+shared: lexer.h hcxselect.o 
+	$(CXX) -shared hcxselect.o -o libhcxselect.so
+
 clean:
 	$(RM) *.o *.a lexer.h
